### DISCUSSION TITLE
Add dark mode to 86Box

### DISCUSSION THREAD

**Discussion by radeon7000** [86Box/86Box#4047] [Ideas]
**Is your feature request related to a problem? Please describe.**
The feature is not related to any problem. The feature is just an enhancement to 86Box's user interface.

**Describe the solution you'd like**
Add dark mode to 86Box's user interface. If possible, update the 86Box manager to also get dark mode.

**Describe alternatives you've considered**
Alternatives are not yet found as of now for 86Box. But using WinBox for the manager solves 86Box manager problem.

**Additional context**
No additional context

**Comment by Cacodemon345** [comment]
Darkmode will come when the UI rewrite is done and merged.

**Comment by Shoegzer** [comment]
Now that dark mode has been implemented for Windows users with v5, would be great to have in Linux as well.

**Reply by OBattler** [reply]
It's already there for Linux and Mac - it just depends on the OS setting (well, it does on Windows as well).

**Comment by Shoegzer** [comment]
That's odd, I just built v5 and I only have light mode, even though my OS setting uses dark mode.  Is there a way to force it?

**Reply by OBattler** [reply]
Strange. Try building it with Qt 6 and see if that fixes it.

**Reply by OBattler** [reply]
But notice: Even there it tends to be behavior specific to the desktop environment used.

**Comment by Shoegzer** [comment]
After building with Qt 6.9.1, It still doesn't show dark mode.  Is it picking up the old configuration or something?
```
$ qmake --version
QMake version 3.1
Using Qt version 6.9.1 in /usr/local/Qt-6.9.1/lib
```
OS: Linux Mint 22.1 Cinnamon with X11 / kernel 6.14

**Comment by Shoegzer** [comment]
I built with:
```
git clone https://github.com/86Box/86Box.git
cmake -B build -S . --preset optimized -GNinja
ninja
```
Here is my [build.log](https://github.com/user-attachments/files/22014674/build.log).

**Comment by Shoegzer** [comment]
What should have been clear to me is the need to specify `-DUSE_QT6=ON` with cmake, otherwise the project defaults to Qt5.  So please disregard, dark mode works just fine in Linux.

Thank you for this incredibly useful and important project.
